docs/package_templates/cmake_package/all/test_package/test_package.cpp

```diff
@@ -5,7 +5,7 @@
 
 int main(void) {
     std::cout << "Create a minimal usage for the target project here." << std::endl;
-    std::cout << "Avoid big examples, bigger than 100 lines" << std::endl;
+    std::cout << "Avoid big examples, or code bigger than 15 lines" << std::endl;
     std::cout << "Avoid networking connections." << std::endl;
     std::cout << "Avoid background apps or servers." << std::endl;
```

Comment:
```suggestion
    std::cout << "Avoid background apps or servers." << std::endl;
    std::cout << "Avoid using binary files in tests." << std::endl;
```
Binary files or extra test files in general have been the most common reason for test_package changes in recent months. Might want to mention it in this message.